### PR TITLE
Added the ability to control how score gets exported

### PR DESCRIPTION
**Pull request by ThinkRedstone**
Again, this is above one commit from sorting icons; otherwise no teams are rendered in ranking.hrml.

### CODE REVIEW

**Comment by idanstark42** [comment]
Will merge this after we merge the sort icon PR

File: src/js/views/ranking.js [hunk 1]
```diff
@@ -102,57 +102,53 @@ define('views/ranking',[
...
             };
 
-            // Rebuild CSV data and filenames when scoreboard is updated
-            $scope.$watch("scoreboard", function() {
-                $scope.rebuildCSV($scores.scoreboard);
+            $scope.$watch("scoreboard", function () {
+               $scope.buildExportFiles();
             }, true);
 
+            $scope.$watchCollection("settings", function () {//we need to rebuild the files if the user changes his export format
```
Comment (idanstark42) [inline src/js/views/ranking.js]: That means you calculate the export files every time the settings change, even if the change has nothing to do with it? Why not $watching something more specific?

File: spec/views/rankingSpec.js [hunk 2]
```diff
@@ -10,32 +10,38 @@ describe('ranking', function() {
...
             $scope = $rootScope.$new();
             scoresMock = createScoresMock($q);
             handshakeMock = createHandshakeMock($q);
-            stagesMock = createStagesMock();
+            stagesMock = createStagesMock($q);
             messageMock = createMessageMock();
+            var settings = {};
+            settings.lineStartString = "\"";
+            settings.lineEndString = "\"";
+            settings.separatorString = "\",\"";
```
[the remark this answers is not in the document]
Comment (ThinkRedstone) [reply spec/views/rankingSpec.js]: Just seemed easier to read by breaking it to multiple lines.

[the remark this answers is not in the document]
Comment (idanstark42) [reply spec/views/rankingSpec.js]: It's okay, doesn't really matter

**Comment by idanstark42** [comment]
Just resolve the conflicts, and we can merge

**Pull request merged**